samples/samples-powershell/AddProductWithIdentityColumn/run.ps1

```diff
@@ -6,7 +9,6 @@ param($Request, $TriggerMetadata)
 # Write to the Azure Functions log stream.
 Write-Host "PowerShell function with SQL Output Binding processed a request."
 
-# Update req_query with the query of the request
 $req_query = [ordered]@{
```

Comment:
For all these [ordered] usages please add a comment with link to docs explaining why this is necessary